### PR TITLE
revamp trend layout of sidebarpanel, chart subtitle and tooltip.

### PR DESCRIPTION
**Pull request by vaelliott**
Trend tab changes should be ready.

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -1303,6 +1303,26 @@ function(input, output, session) {
...
   #### Time trend plot ----
   ###############################################.  
+  # Trend help pop-up
+  observeEvent(input$help_trend, {
+    
+    showModal(modalDialog(
+      title = "How to use this chart",
+      p("The trend chart is designed to explore how a single indicator has changed over time for one or more geograpical areas.
+               First select an indicator then add one or more geographical area to the chart using the geography filters in 'Step 2'"),
+      p(column(7,img(src="help_trend_chart2.png")),
```
Comment (jvillacampa) [inline server.R]: You haven't commited/pushed the file help_trend_chart2.png

File: server.R [hunk 2]
```diff
@@ -1303,6 +1303,26 @@ function(input, output, session) {
...
+        column(5,
+               p("You can add more than one area/area type to the trend chart, it is possible to show a mixture of areas like council area and NHS board."),
+               p("Occasionally a complete time series data are not available."),
```
Comment (jvillacampa) [inline server.R]: This sentence sounds odd

File: ui.R [hunk 3]
```diff
@@ -52,7 +52,7 @@ tagList( #needed for shinyjs
                           .col-md-11, .col-lg-11, .col-xs-12, .col-sm-12, .col-md-12, .col-lg-12 {
                           padding-left: 5px; padding-right: 5px;}",
                           #Style for download buttons
-                          ".down{background-color:#4da6ff; color: white; background-image:none; min-width: 23vh;
+                          ".down{background-color:#4da6ff; color: white; background-image:none; min-width: 20vh;
```
Comment (jvillacampa) [inline ui.R]: So, this causes a slight issue as the "Download data" boxes become wider than the rest (see rank tab). We can talk about it,

Comment (jvillacampa) [reply ui.R]: Maybe we could solve it changin it to "Save data"

File: ui.R [hunk 4]
```diff
@@ -289,18 +289,19 @@ tabPanel("Summary", icon = icon("list-ul"), value = "summary",
...
-                          awesomeCheckbox("ci_trend", label = "95% confidence intervals", value = FALSE)),
-                      actionButton("defs_trend", label="Definitions", icon= icon('info'), class ="down"),
+                      column(6,
+                             actionButton("help_trend",label="Help", icon= icon('question-circle'), class ="down")),
+                      column(5,
+                             actionButton("defs_trend", label="Definitions", icon= icon('info'), class ="down")),
+                      column(12,
+                      shiny::hr(),
+                      div(title="Select an indicator to see trend information. Click in this box, hit backspace and start to type if you want to quickly find an indicator.",
+                          selectInput("indic_trend", "Step 1. Select an indicator", choices=indicator_list)),
```
Comment (jvillacampa) [inline ui.R]: Should we include in the label as well that they can start typing to get it. Something like: "Step 1. Select and indicator (hit backspace and start typing to search)"

**Comment by jvillacampa** [comment]
Also, I have noticed that the legend label for localities it's pretty odd. It includes a lot of white of space before having the are type indicator (e.g. Abeerdeen Central                                            - HSCL). I think this issue is related with the data preparation or with the geography lookup. Would you mind taking a look at this?

File: server.R (hunk 2 above)
Comment (vaelliott) [reply server.R]: text revised

File: ui.R (hunk 3 above)
Comment (vaelliott) [reply ui.R]: reverted to min-width 23vh simplest solution for now

Comment (vaelliott) [reply ui.R]: reverted to min-width 23vh simplest solution for now

File: ui.R (hunk 4 above)
Comment (vaelliott) [reply ui.R]: adjusted select input title to include text about typing to find indicator

**Pull request merged**